### DISCUSSION TITLE
prese che non usano cloud?!

### DISCUSSION THREAD

**Discussion by 50494554524F** [indomus/forum#422] [HUB: Home Assistant]
avevo cominciato a leggere riguardo le prese smart e per non spendere (e non studiare) su zigbee - visto che è più per sfizio/imparare - ho dedotto che la soluzione
giusta sia usare prese wifi/bluetooth. in particolare, prese tuya cloni su cui caricare tasmota., economiche.
 ultimo check sul sito di tasmota, tuya ha modificato il firmware ed ora non è più possibile caricare il loro (o bisognerebbe trovare prese non ancora aggiornate, difficile).
che faccio ora?! consigli?! marche?! non mi fate ricominciare a leggere tutto da capo :D
grazie

**Comment by snakuzzo** [comment]
Ok...quello che hai scritto è tutto sbagliato.
1. La soluzione "giusta" non esiste.
2. il firmware Tasmota (o altri) non lo carichi solo sulle prese TUYA
3. Prese ottime esistono anche Zigbee
4. Non è che il firmware Tuya è cambiato...è che molti produttori non utilizzano più chip esp826x ma realtek

Consiglio...
Non so dove tu abbia letto quello che hai affermato, ma credo sia il caso di informarsi meglio

**Reply by 50494554524F** [reply]
1. giusta per me, costano pure 5€, erano perfette :D
2. certo, ma dovrò rileggere tutto perchè viste le tuya non mi sono appuntato niente
3. come ho scritto, a parte spenderci altri soldi solo per sfizio mi toccherebbe studiare un sacco di altra roba quando devo finire ancora di padroneggiare home assistant
4. è scritto nella pagina di una qualsiasi delle varie "tuya" .... https://templates.blakadder.com/BSD33-16A.html ... segui il link e leggi del problema di firmware

**Reply by snakuzzo** [reply]
4. Se segui il link indicato in guida...vedrai che parla del discorso del chip alla fine ;)

**Reply by 50494554524F** [reply]
anche il chip... ma il problema del firmware è qui.... https://github.com/ct-Open-Source/tuya-convert/wiki/Collaboration-document-for-PSK-Identity-02

è più di un anno che c'è il problema, quindi, anche comprandole con esp si rischia di non poterle usare... non capisco 'sta fissa di obbligare all'uso del cloud

vabbuò, prendo coraggio e stasera ricomincio a leggere da capo... cmq suggerite se ne usate di wifi offline

**Reply by snakuzzo** [reply]
io come WIFI uso le Blitzwolf SHP2...ma anche lì te la rischi con il discorso del chip

**Comment by 50494554524F** [comment]
per non impazzire, cambio i termini... prese che funzionano anche offline... sennò si ripresenterà il problema per i pulsanti etc etc...

abbandono l'idea tasmota... al cloud tuya gli affianco l'integrazione local tuya, analogamente a quanto fatto per tado... per adesso preferenza per le aunics shucko + italia o una dei tanti cloni solo shucko da 5€ su aliexpress... e tuya è sistemata

ora diventa più semplice e devo solo ricontrollare le integrazioni per vedere quelle che funzionano anche offline o trovare aggeggi compatibili homekit

lo studio continua... (non voglio copiare cose già fatte e vi rompo mentre penso :D )

**Reply by 50494554524F** [reply]
ho comprato aunics in offerta per provare, local tuya funziona... quindi, visto che i "tuya" costano meno degli altri, mi comprerò le semplici shucko ed il resto su aliexpress

**Comment by KC-inDomus** [comment]
uso ottimamente [Shelly Plug S](https://indomus.it/componenti/shelly-plug-s/). localmente, via mqtt.

**Reply by 50494554524F** [reply]
grazie, ormai vado di tuya... basta usare yaml

**Comment by 50494554524F** [comment]
se avete dei "tuya" iscrivetevi al programma beta della integrazione ufficiale che vi regalano un buono da $750 ... devo solo capire che ci si può fare :D

**Comment by 50494554524F** [comment]
sono stato fortunato... tuya ufficiale supporterà anche la modalità offline... 

ora c'è anche l'integrazione meross (hacs) per offline ... la scelta si amplia per chi vuol spendere poco

**Reply by Scialla** [reply]
Io ti consiglio di non legarti a custom component, e pure le integrazioni a volte tardano ad aggiornarsi. Sicuramente è meglio fare da se con mqtt ed, eventualmente, zigbee2mqtt.
I device zigbee li compri in Cina, Aqara, Mija, Blitzwolf, e costano pure meno..

Poi fai cosa ti piace di più, ma se ascolti chi ci è già passato..
